### PR TITLE
txtincLAB2

### PR DESCRIPTION
**Pull request by txtincorporated**
This is for LAB2; LAB1 still needs tests

### CODE REVIEW

**Comment by GeoffreyEmerson** [comment]
It's a Sander error. On your local machine, you've got those directories in place, but on the test environment you're telling sander to read a directory that doesn't exist.

**Comment by txtincorporated** [comment]
Schweet!  Thx, Geoff.

File: index.js [hunk 1]
```diff
@@ -1,4 +0,0 @@
-const server = require('./lib/server');
-
-console.log('Calling server...');
-server.start();
```
Comment (GeoffreyEmerson) [inline index.js]: Your tests pass, but for some reason you deleted your index.js so you now have no way to run your app.

File: lib/server.js [hunk 2]
```diff
@@ -4,10 +4,7 @@ const app = require('./app');
 
 function start() {
 
-  console.log('server starting...');
-
   http.createServer(app).listen(3000, () => {
-    console.log('Server has started on port 3000.');
   });
   
 }
```
Comment (GeoffreyEmerson) [inline lib/server.js]: This is way more confusing than it needs to be. This file should be in your project root, and it should not wrap anything in a function. Check out this example:
```
const http = require('http'); // Include node's http functions
const app = require('./lib/app'); // Grab your express routing object
const port = process.env.PORT || 3000; // Get the port number or default

const server = http.createServer(app); // Make your express routes into a real server
server.listen(port, () => { // Begin listening for http requests
  console.log('server running at', server.address()); // Log to console that server has started
});
```

**Comment by txtincorporated** [comment]
11/17/16  

Should be good now, Geoff.

—NPK


Nathan Keene
npk@txtincorporated.com
(646)675-5694

**Comment by GeoffreyEmerson** [comment]
Looks good!

**Pull request merged**